stellarphot/core.py

```diff
@@ -565,35 +579,21 @@ class PhotometryData(BaseEnhancedTable):
         "file": None,
     }
     observatory = None
-    camera = None
+    camera = TableAttribute(default=None)
 
     def __init__(
         self,
         *args,
         input_data=None,
         observatory=None,
-        camera=None,
         colname_map=None,
         passband_map=None,
         retain_user_computed=False,
         **kwargs,
     ):
-        if (observatory is None) and (camera is None) and (input_data is None):
+        if (observatory is None) and (self.camera is None) and (input_data is None):
             super().__init__(*args, **kwargs)
         else:
-            # Perform input validation
-            if not isinstance(observatory, EarthLocation):
-                raise TypeError(
-                    "observatory must be an "
-                    "astropy.coordinates.EarthLocation object instead "
-                    f"of type {type(observatory)}."
-                )
-            if not isinstance(camera, Camera):
-                raise TypeError(
-                    "camera must be a stellarphot.Camera object instead "
-                    f"of type {type(camera)}."
-                )
-
             # Check the time column is correct format and scale
```

Comment:
Caught a typo in line 606 of code, comment should be
`# Happens if first item doesn't have a "scale"`
not
`# Happens if first item dosn't have a "scale"`